src/test/java/picard/util/LiftoverVcfTest.java

```diff
@@ -1211,11 +1208,11 @@ public void testUnreadableDictionary() throws IOException {
                 "CHAIN=" + CHAIN_FILE,
                 "REFERENCE_SEQUENCE=" + REFERENCE_FILE,
         };
-        final Path dictionary = REFERENCE_FILE.toPath().resolveSibling(REFERENCE_FILE.getName().replace("fasta", "dict"));
+        final Path dictionary = REFERENCE_FILE.toPath().resolveSibling(REFERENCE_FILE.getName().replaceAll("fasta$", "dict"));
```

Comment:
I like the solution you did above (of copying the data you need) much better than modifying the test files and then changing them back in a finally clause...could you also do it here?